docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+  in newer versions of Django or Python.
+
+
+Major Changes
+=============
+
+These are by no means exhaustive, check the `git log
+<https://github.com/translate/pootle/compare/stable%2F2.5.1...master>`_
+for more details.
+
+
+- Statistics calculations:
+
+  - Major rewrite of the way statistics are handled. RQ is used to manage the
+    calculation of invalidated statistics.  On the frontend side, stats are now
+    loaded asynchronously, thus any pending calculations no longer block page
+    loads.
+  - Custom word counting calculation method.
+
+
+- Big revamp of UI:
+
+  - We now have a wide stats table that shows the same data in the different
+    levels. This affects several features which are now relocated in a sidebar:
+    actions, instructions or current goal.
+  - New navigation scheme using dropdowns.
+  - Removed zoom feature. Now users should rely on their browsers.
+  - *All Projects* page now shares the same browsing table as available in any
+    other overview page.
+  - When there are failing checks, overview tables now display the number of
+    units which have failing checks, not the total number of failing checks.
+  - Table sorting is now remembered across overview pages, and not separately
+    in project, language and translation project pages.
+  - Major speed improvements when calculating last action information.
+  - User actionable items in the navbar have been moved to a drop-down.
+  - When selecting languages, redirect logic is smarter now.
+  - Different last activity messages for new translations and edits.
+  - Added landing page for anonymous users.
+  - The welcome page displays the top scorers over the last 30 days.
+  - User score is displayed in the header and dynamically updated when
+    translation actions are made.
+  - Several layout improvements that take into account different screen sizes.
+  - Filters allow sorting units according to their last action date.
+  - New navigation scheme:
+  
+    + All directories/files for a project are displayed in a new drop-down.
+      The differences between directories and files are visually highlighted.
+    + Directories/files can be navigated and translated across multiple
+      languages in a project.
+    + Tabs have been replaced in favor of drop-down menus.
+    + The editor search box is now displayed in the action links section,
+      keeping its positioning consistent with the overview page.
+    + A new action link in the editor, *Go back to overview*, allows users to
+      go back to the same place they entered translation mode from.
+
+  - Overview pages now report the last time a unit was added to a
+    store/project. In the browsing tables a *Last Updated* column is displayed
+    and in the extended stats *Created* and *Last Updated* dates.
+  - Implemented project notifications by reusing static pages. Notifications
+    are per-project and are displayed across languages (automatically adapting
```

Comment:
For now this is Ok. The alterations for allowing to save announcements for TPs and Languages, and display those, are not yet finished nor merged.